core/ArchiveProcessor/PluginsArchiver.php

```diff
@@ -68,6 +68,7 @@ public function __construct(Parameters $params, ArchiveWriter $archiveWriter = n
         $this->archiveWriter->initNewArchive();
 
         $this->logAggregator = new LogAggregator($params);
+        $this->logAggregator->allowUsageSegmentCache();
```

Comment:
We only do it during archiving... if other usages want to make use of it, they need to enable it as well... prevents performing the create temporary table logic for some live requests like Heatmaps where it otherwise would possibly apply the segment on a yearly range etc.

Reply:
Also prevents that someone maybe doesn't call the cleanup command etc

Reply:
Also prevents that we possibly create heaps of temporary tables depending on the ui/api request...